pyiron_base/generic/datacontainer.py

```diff
@@ -152,6 +152,18 @@ class DataContainer(MutableMapping):
     [3, 2, 1, 0]
     >>> list(pl.keys())
     [0, 1, 2, 3]
+
+    .. attention:: Subclasses beware!
```

Comment:
Can you double check this is the right docstring format? I thought we used `Attention:` rather than `.. attention::`

Reply:
This is reST specific syntax to get rendered as a little pop up box, see the [pre-render](https://pyiron-base.readthedocs.io/en/docs/apidoc/pyiron_base.generic.datacontainer.html).  There's actually a small issue with the list inside not getting properly rendered.